.github/workflows/rust.yml

```diff
@@ -62,15 +62,14 @@ concurrency:
 jobs:
   check:
     name: check
-    runs-on:
-      group: 4-cores_16GB_Ubuntu Group
+    runs-on: buildjet-4vcpu-ubuntu-2204
     timeout-minutes: 60
     steps:
       - uses: actions/checkout@v3
       - uses: rui314/setup-mold@v1
       - name: Install Rust
         run: rustup show
-      - uses: Swatinem/rust-cache@v2
+      - uses: astriaorg/buildjet-rust-cache@v2.5.1
```

Comment:
I wonder if we can continue using Swatinem's action by using the new `buildjet` backend – I'm not sure it's equivalent, but it looks like something was upstreamed (https://github.com/Swatinem/rust-cache/pull/154) so it's worth a try. Like this:

```yml
- uses: Swatinem/rust-cache@v2
  with:
    # ...
    cache-provider: "buildjet"
```

Reply:
Nice! I didn't see that his had been upstreamed already, I'll try switching back